infrastructure/builder/src/build/service.js

```diff
@@ -24,7 +17,6 @@ const generateServiceTasks = ({tasks, baseDir, spec, cfg, name, cmdOptions}) =>
     break;
 
   case 'web-ui':
-    assert(!isMonorepo, 'monorepo not supported for this buildtype'); // TODO build from monorepo
     webUiTasks({tasks, baseDir, spec, cfg, name, cmdOptions, repository, workDir});
     break;
```

Comment:
This whole stanza can go now, I think?  Along with `service/web-ui.js`.  `web-ui` never appears in the spec.